packages/lime-full/Makefile

```diff
@@ -22,16 +22,16 @@ define Package/$(PKG_NAME)
   CATEGORY:=LiMe
   MAINTAINER:=Pau Escrich <p4u@dabax.net>
   URL:=http://libre-mesh.org
-  DEPENDS:=+lime-system +lime-proto-bmx6 +lime-proto-batadv +lime-proto-anygw +lime-proto-wan \
+  DEPENDS:=+lime-system +lime-proto-olsr +lime-proto-olsr6 +lime-proto-batadv +lime-proto-anygw +lime-proto-wan \
```

Comment:
Is ok to make *lime-full* depending from just one layer 3 protocol or it should depend on the *full* list of supported protocols, now that this wouldn't hurt?